app/models/User.coffee

```diff
@@ -85,6 +86,29 @@ module.exports = class User extends CocoModel
   isSessionless: ->
     Boolean((utils.getQueryVariable('dev', false) or me.isTeacher()) and utils.getQueryVariable('course', false) and not utils.getQueryVariable('course-instance'))
 
+  getClientCreatorPermissions: ->
+    clientID = @get('clientCreator')
+    if !clientID
+      clientID = utils.getApiClientIdFromEmail(@get('email'))
+    if clientID
+      api.apiClients.getByHandle(clientID)
+      .then((apiClient) => 
+        @clientPermissions = apiClient.permissions
+      )
+      .catch((e) =>
+        console.error(e)
+      )
+
+  canManageLicensesViaUI: ->
```

Comment:
Could this be `canManageLicensesViaUI: -> @clientPermissions?.manageLicensesViaUI ? true`

Reply:
Yes. Will update.